src/mesh/voxel_cloud.rs

```diff
@@ -1200,13 +1262,98 @@ impl ScalarField {
                     }
                 }
             }
-
-            // Process the current voxel. If it is outside the volumes, set its
-            // value to be positive, if it's inside, set it to negative.
-            self.voxels[one_dimensional] = if discovered_as_outer_and_empty[one_dimensional] {
-                Some(distance)
-            } else {
-                Some(-distance)
+            let is_outside = discovered_as_outer_and_empty[one_dimensional];
+
+            // Process the current voxel.
+
+            self.voxels[one_dimensional] = match falloff_function {
+                FalloffFunction::Linear(mul) => {
+                    // distance values: 0 to infinity
+                    // suggested multiplier: 1
+                    // output values: 0 to infinity
+                    // value on volume: 0
```

Comment:
This information is great! But it would be even greater if it wasn't an internal comment inside a function, but an actual doc comment on `FalloffFunction::Linear`. Same goes for all other variants of `FalloffFunction`.